add process_terminal() and update links to other functions

arlenk opened the pull request:
updated existing links and added reference to process_terminal()

If it's desirable, I can also make these canonical links (https://help.github.com/articles/getting-permanent-links-to-files/) so they don't need to be updated in the future.  The downside, though, is that they will always link to a static revision of these files.

drashna commented:
Yes, canonical links would be better, actually.  Since these are likely to change in the future, it would be best, long term.

drashna commented:
Also, should probably add `process_clicky` in there, as well.

arlenk commented:
> Yes, canonical links would be better, actually. Since these are likely to change in the future, it would be best, long term.

ok, changed the links to permalinks and added a few other process_* functions that I missed the first time (process_clicky, process_steno, etc.)
